ios-trivia-game/ViewControllers/ResultsViewController.swift

```diff
@@ -32,4 +33,15 @@ class ResultsViewController: UIViewController {
     }
     */
 
+    @IBAction func onQuitFromResults(_ sender: UIBarButtonItem) {
+        // update user_in_game to remove player from user_in_game
+        
+        FirebaseClient.instance.quitGame(complete: {
```

Comment:
That's kind of what quitGame is for.

the complete and error handlers should be implemented by the callers separately